modules/standard/BigInteger.chpl

```diff
@@ -4044,38 +4066,69 @@ When ``n/d`` does not produce an integer, this method may produce incorrect resu
         const d_ = d;
 
         select rounding {
-          when Round.UP   do mpz_cdiv_q(this.mpz, n_.mpz, d_.mpz);
-          when Round.DOWN do mpz_fdiv_q(this.mpz, n_.mpz, d_.mpz);
-          when Round.ZERO do mpz_tdiv_q(this.mpz, n_.mpz, d_.mpz);
+          when round.up   do mpz_cdiv_q(this.mpz, n_.mpz, d_.mpz);
+          when round.down do mpz_fdiv_q(this.mpz, n_.mpz, d_.mpz);
+          when round.zero do mpz_tdiv_q(this.mpz, n_.mpz, d_.mpz);
           }
       }
     }
   }
 
+  deprecated
+  "bigint.div_q using Round is deprecated, use bigint.div_q with round instead"
+  proc bigint.div_q(const ref n: bigint,
+                              d: integral,
+                    param     rounding: Round) {
+    use Round;
+    if (rounding == UP) {
+      this.div_q(n, d, round.up);
+    } else if (rounding == ZERO) {
+      this.div_q(n, d, round.zero);
+    } else {
+      this.div_q(n, d, round.down);
+    }
+  }
+
   proc bigint.div_q(const ref n: bigint,
                               d: integral,
-                    param     rounding = Round.ZERO) {
+                    param     rounding = round.zero) {
 
     this.div_q(n, new bigint(d), rounding);
   }
 
+  deprecated
+  "bigint.div_r using Round is deprecated, use bigint.div_r with round instead"
   proc bigint.div_r(const ref n: bigint,
                     const ref d: bigint,
-                    param     rounding = Round.ZERO) {
+                    param     rounding: Round) {
```

Comment:
Why was this changed from Round.ZERO to Round?

Reply:
See OP:

> When deprecating, replace the default argument value with just a type
declaration to avoid potential conflicts when relying on the default value (such
calls will naturally switch to using the new type)